assets/css/style.scss

```
@@ -34,10 +34,14 @@ main.page-content {
   color: #111;
 }
 
+.table-borderless, .table-borderless *, .table-borderless * * {
+  border-style : hidden !important;
+  background-color: rgba(0,0,0,0) !important;
+}
+
```

Probably not the most elegant way to accomplish this, but something like:

```suggestion

table.preamble > tbody > tr > :not(:first-child) {
  width: 100%;
}

table.preamble > tbody > tr > :first-child {
  white-space: nowrap;
}
```

---

What does this do?